plugins/services/src/js/reducers/serviceForm/common/Constraints.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
         return memo;
       }
 
-      if (item.error) {
+      if (item instanceof Error) {
         memo.push(
```

Comment:
Does it make sense to push the actual Error message instead of `"value-is-malformed"` on the line 81?

Reply:
Could be, but then I would say that the error message should be "value-is-malformed". And I would split the consistency of messages across files. 

I would propose to use the same name on `new Error`, and the message who is sent, but not to expose error.msg

Reply:
Hum, changed my mind... will return item.message